src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
+            /// </summary>
+            page.StandardKoTMode();
+            page.KOTWindowSizeWithinRange();
+            Size smallKOTWindowSize = new Size(161, 168);
+            Size largeKOTWindowSize = new Size(502, 502);
+            Size mediumKOTWindowSize = new Size(396, 322);
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallKOTWindowSize;
+            Assert.AreEqual("(161, 168)", page.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeKOTWindowSize;
+            Assert.AreEqual("(502, 502)", page.GetCalculatorWindowSize());
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = mediumKOTWindowSize;
+            Assert.AreEqual("(396, 322)", page.GetCalculatorWindowSize());
+            page.StandardNonKoTMode();
+        }
+
+        [TestMethod]
+        [Priority(2)]
+        public void KOT_ScaleRetention()
+        {
+            /// <summary>
+            /// (Half) Test Case 23458956: BVT: KOT: Verify scale retention
+            /// This automated test verifies the window scale retention is maintained when switching between KOT and Non-KOT
+            /// Note: This does not verify scale retention across calculator session
+            /// </summary>
+
+            Size smallWindowSize = new Size(464, 502);
+            Size largeKOTWindowSize = new Size(502, 502);
+            
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = smallWindowSize;
+            Assert.AreEqual("(464, 502)", page.GetCalculatorWindowSize());
+
+            page.StandardKoTMode();
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeKOTWindowSize;
+            Assert.AreEqual("(502, 502)", page.GetCalculatorWindowSize());
+
+            page.StandardNonKoTMode();
+            Assert.AreEqual("(464, 502)", page.GetCalculatorWindowSize());
+
+            page.StandardKoTMode();
+            Assert.AreEqual("(502, 502)", page.GetCalculatorWindowSize());
+
+        }
+
+        [TestMethod]
+        [Priority(2)]
+        public void KOT_ErrorMessage()
+        {
+            /// <summary>
+            /// Test Case 23459118: BVT: KOT: Verify error message display
+            /// This automated test verifies the window scale retention is maintained when switching between KOT and Non-KOT
+            /// Note: This test case is in progress: Need to verify " All operator buttons (except for "CE", "C", "Delete" and "="), and memory buttons are disabled"
+            /// </summary>
+
+            Size largeKOTWindowSize = new Size(502, 502);
+            Point largeKOTWindowPosition = new Point(0, 0);
+            page.StandardKoTMode();
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Position = largeKOTWindowPosition;
+            WinAppDriver.Instance.CalculatorSession.Manage().Window.Size = largeKOTWindowSize;
+            page.Window.SendKeys("/");
```

Comment:
Send keys are not self-explanatory so they need comments. Please add comments for any send keys that are not obvious